recipes/snp-pipeline/meta.yaml

```diff
@@ -33,6 +31,13 @@ test:
   requires:
     - testfixtures
 
+  commands:
+    - cfsan_snp_pipeline --version
+
+extra:
```

Comment:
Try to remove `setuptools` from run reqs. It is normally only needed during build step.

Reply:
Please don't as it it really required: https://github.com/CFSAN-Biostatistics/snp-pipeline/blob/v1.0.1/snppipeline/data.py#L12.

Reply:
thanks @mbargull my mistake! I missed it used `pkg_resources`